appengine/flexible/websockets/main.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2017 Google Inc.
```

Comment:
**2018** Google **LLC**?

Apply this comment throughout your PR, if you make this change.

Reply:
Fixed